templates/taxi/index.html

```diff
@@ -10,4 +10,5 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+    <p>You have visited this page {{ num_visits }} time{{num_visits|pluralize}}</p>
 {% endblock %}
```

Comment:
Make sure to use 2 whitespaces indentations in your `.html` files as per the checklist requirement.